src/components/RecoveryWords.js

```diff
@@ -123,18 +130,19 @@ class RecoveryWords extends Nimiq.Observable {
 
     async _checkPhraseComplete() {
         // Check if all fields are complete
-        if (this.$fields.some(field => !field.complete)) {
+        if (this.$fields.some(field => 'complete' in field && !field.complete)) {
             this._onFieldIncomplete();
             return;
         }
 
         try {
-            const mnemonic = this.$fields.map(field => field.value);
+            const mnemonic = this.$fields.map(field => ('value' in field ? field.value : '-'));
             const type = Nimiq.MnemonicUtils.getMnemonicType(mnemonic); // throws on invalid mnemonic
             this._mnemonic = { words: mnemonic, type };
             this.fire(RecoveryWords.Events.COMPLETE, mnemonic, type);
         } catch (e) {
-            if (e.message !== 'Invalid checksum') {
+            const err = /** @type {Error} */ (e);
+            if (err.message !== 'Invalid checksum') {
                 console.error(e); // eslint-disable-line no-console
```

Comment:
```suggestion
                console.error(err); // eslint-disable-line no-console
```

_Super picky comment_

I wouldn't use `e` after defining `err` with a type